vars/runSmokeTest.groovy

```diff
@@ -157,7 +157,7 @@ private def runPipeline(String project, String ocDeployerBuilderPath, String ocD
             export DYNACONF_OCPROJECT=${project}
 
             set +e
-            iqe tests plugin advisor --junitxml=junit.xml -s -v -m ${pytestMarker} --log-file=iqe.log --log-file-level=DEBUG 2>&1 | tee pytest-stdout.log
+            iqe tests plugin ${plugin} --junitxml=junit.xml -s -v -m ${pytestMarker} --log-file=iqe.log --log-file-level=DEBUG 2>&1 | tee pytest-stdout.log
```

Comment:
`plugin` is not a defined variable at this point in the code so this would not work for you.

If you see above there is a TODO line to switch back to using 'iqe tests all' -- I can work on that today. The only reason I took that out is because we had some test collection issues in the iqe framework several weeks ago.